_layouts/tutorial.html

```diff
@@ -0,0 +1,24 @@
+---
+layout: default
+---
+
+<dl>
+  <dt>Level</dt>
+  <dd>{{ page.level }}</dd>
```

Comment:
Level also appears a second time below; remove these lines?

Reply:
Thanks for catching this! I deleted the first _Level_ field in tutorial.html.